src/index.html

```diff
@@ -11,7 +11,176 @@
+          <label class="form_field">
+            Name:
+            <input
+              type="text"
+              name="name"
+              maxlength="20"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            How old are You?
+            <input
+              type="number"
+              name="number"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <div>
+          <label class="form_field">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input
+              type="password"
+              name="pwd"
+              minlength="8"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="field">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="form_field" >
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="loveOfACats"
+            value="Yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+           type="radio"
+           id="no"
+           name="loveOfACats"
+           value="No"
+          >
+          <label for="no">No</label>
+        </div>
+        <div>
+          <label class="form_field">
+            What is your favorite color?
+            <input
+              type="color"
+              name="favcolor"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div>
```

Comment:
Add the 'form_field' class to the label element for consistency with other labels.